HaxeManual/06-language-features.tex

```diff
@@ -165,12 +165,25 @@ \chapter{Language Features}
 
 \begin{lstlisting}
 class Main<A> {
-    static function main() {
-        new Main<String>("foo");
-        new Main(12); // use type inference
-    }
+  static function main() {
+    new Main<String>("foo");
+    new Main(12); // use type inference
+  }
 
-    function new(a:A) { }
+  function new(a:A) { }
+}
+\end{lstlisting}
+
+\emph{\tref{Markup Literals}{lf-markup}:}
```

Comment:
Let's not add this for now.

Reply:
The short summary on the LF page or the entire section?

Reply:
I'd hold back on the entire section for now because Nicolas still wants to change something.